src/games/nogo_game.cc

```diff
@@ -5,15 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-#pragma once
 #include "nogo_game.h"
 
 NoGoGame::NoGoGame() {
   Reset();
 }
-NoGoGame::~NoGoGame() {
```

Comment:
at other times you have inserted (virtual) destructors; now you remove a destructor. What's the logic?

Reply:
I prefer to remove the destructor if we just want the default destructor.

However if the class is virtual (i.e. has any virtual members) then its destructor has to be `virtual` and this is not done by the compiler by default. Else we set the nasal demons free.

See comment in https://github.com/AlignmentResearch/polygames/blob/eliminate_warnings/src/mcts/mcts.h#L70

and stackoverflow https://stackoverflow.com/a/10024812

I discovered this annoying fact via a Clang compiler warning.